.github/workflows/tethys.yml

```diff
@@ -0,0 +1,161 @@
+        uses: actions/checkout@v2
+      # Build the docker for no tag
+      - name: Build Without Tag
+        run: |
+          docker build -t ${{ secrets.DOCKER_UPLOAD_URL }}:master .;
+      # Upload docker if pull request no tag
+      - name: Upload Docker No Tag
+        if: ${{ github.event_name != 'pull_request' }}
+        run: |
+          echo "Pushing to docker registry";
+          echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin;
+          docker push ${{ secrets.DOCKER_UPLOAD_URL }}:master;
+      # No Upload if Pull Request
+      - name: No Upload
+        if: ${{ github.event_name == 'pull_request' }}
+        run: |
+          echo "Uploading is skipped for pull requests."
+
+  tethys-conda:
+    name: Tethys Conda (${{ matrix.platform }})
+    needs: tethys-tests
+    if:
+      contains('
+      refs/heads/master
+      refs/heads/release
+      ', github.ref)
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.7]
+    steps:
+      # Checkout the source
+      - name: Checkout Source
+        uses: actions/checkout@v2
+      # Setup Tethys
+      - name: Setup Tethys
+        run: |
+          cd ..
+          bash ./tethys/scripts/install_tethys.sh --partial-tethys-install me -n tethys -s $PWD/tethys
+          . ~/miniconda/etc/profile.d/conda.sh
+          conda activate tethys
+          hash -r
+          conda config --set always_yes yes --set changeps1 no
+          conda update -q conda
+          conda install conda-build anaconda-client
+          conda config --set anaconda_upload no
+          mkdir -p ~/conda-bld
+      # Export Conda Build Path
+      - name: Set Conda Build Path
+        uses: allenevans/set-env@v2.0.0
+        with:
+          CONDA_BLD_PATH: '/home/runner/conda-bld'
+      # Generate Conda Recipe Without Constrained Dependencies
+      - name: Generate Conda Recipe
+        run: |
+          cd ..
+          . ~/miniconda/etc/profile.d/conda.sh;
+          conda activate tethys;
+          tethys gen metayaml --overwrite;
+      # Show Tethys Meta
+      - name: Show Tethys Meta
+        run: |
+          cd ..
+          cat ./tethys/conda.recipe/meta.yaml
+      # Build Conda
+      - name: Build Conda
+        run: |
+          cd ..
+          . ~/miniconda/etc/profile.d/conda.sh;
+          conda activate tethys;
+          conda build -c tethysplatform -c conda-forge ./tethys/conda.recipe
+      # Upload Conda No Pull Request No Tag
+      - name: Upload Conda No Tag
+        if: ${{ github.event_name != 'pull_request' }}
+        run: |
+          cd ..
+          . ~/miniconda/etc/profile.d/conda.sh;
+          conda activate tethys;
+          ls ~/conda-bld/noarch
+          anaconda -t "${{ secrets.CONDA_UPLOAD_TOKEN }}" upload -u aquaveo -l dev $CONDA_BLD_PATH/noarch/tethys-platform*.tar.bz2 --force;
```

Comment:
The user aquaveo is hard coded. Swap it out for an environment variable: CONDA_UPLOAD_USER

Reply:
Done